src/runtime_src/core/edge/skd/xrt_skd.cpp

```diff
@@ -227,6 +227,13 @@ namespace xrt {
     char sk_fini[PS_KERNEL_NAME_LENGTH+finiExtension.size()];
     int ret = 0;
 
+    // Check if SCU is still in running state
+    // If it is, that means it has crashed
+    if((args_from_host[0] & 0x1) == 1) {
+      args_from_host[0] = 2;
```

Comment:
Magic numbers.  It would be helpful for a comment describing what is being done and why 2 and -1 are special.